modules/monitoring/manifests/checks.pp

```diff
@@ -58,6 +59,36 @@
 
   include icinga::plugin::check_http_timeout_noncrit
 
+  $content_data_api_hostname    = "content-data-api.${app_domain}"
+  $content_data_api_metrics_url = '/healthcheck/metrics'
+  $content_data_api_search_url = '/healthcheck/search'
+
+  if $::aws_migration {
+    include icinga::client::check_json_healthcheck
+
+    icinga::check { "content_data_api_search_${::hostname}":
+      check_command              => "check_app_health!check_json_healthcheck!443 ${content_data_api_search_url} ${content_data_api_hostname} https",
+      service_description        => 'search healthcheck for content-data-api',
+      use                        => 'govuk_urgent_priority',
+      host_name                  => $::fqdn,
+      notes_url                  => monitoring_docs_url(content-data-api-app-healthcheck-not-ok),
```

Comment:
I guess this makes sense for now: https://docs.publishing.service.gov.uk/manual/alerts/content-data-api-app-healthcheck-not-ok.html

Should we have a follow-up PR here and in the Dev Docs to split out into two notes_url pages?

Reply:
Indeed.  I'd planned to update the documentation as part of https://trello.com/c/Wn4w66dE, prior to removing the alerts from the instance healthcheck.